src/HawksearchVue.js

```diff
@@ -236,8 +236,7 @@ class HawksearchVue {
 
         var config = store.state.config;
         var params = Object.assign({}, searchParams, { ClientGuid: config.clientGuid, IndexName: config.indexName, DisplayFullResponse: true });
-
-        Vue.http.post(this.getFullAutocompleteUrl(store), params, {
+        axios.post(this.getFullAutocompleteUrl(store), params, {
```

Comment:
The suggestion request needs to be canceled at subsequent suggestion requests. Cancelation van be don like this https://www.npmjs.com/package/axios#cancellation